### DISCUSSION TITLE
Error log notifications

### DISCUSSION THREAD

**Discussion by angieherrera** [solspace/craft-freeform#515] [Questions & Answers] [answered]
Hi! A client is seeing errors in the error log, almost all being authentication errors. We're wondering if there's a way for Freeform/Craft to send an email when an authentication failure happens so the client can update it immediately? Not sure that "Send Email Alerts for Failed Notifications" would be enough, but perhaps I'm wrong?
<img width="1414" alt="CleanShot 2022-11-23 at 13 01 36" src="https://user-images.githubusercontent.com/442434/203646540-ebc9b756-76fb-4ecf-b85b-c1025e1b6a5a.png">

**Comment by kjmartens** [comment, marked as answer]
Hi @angieherrera,

This error is likely caused by a password no longer being valid. As for alerting the site owner (or developer), there is the [Digest](https://docs.solspace.com/craft/freeform/v4/reliability/digest.html) feature, which offers the ability to send the error logs (and other things) via email every day or once a week. There isn't an option to have it sent immediately upon the site receiving an error logged, but daily would be the closest (and it'll send every day regardless of whether there are any errors or not). 🙂

**Reply by angieherrera** [reply]
Excellent, thanks @kjmartens !

**Reply by kjmartens** [reply]
You're welcome! I've noted this as a feature request as well (immediate notifications for issues like this) and we'll see if it's something we can tackle in the near future. 🙂
